### PR TITLE
Gtm/mpas plot external mesh properties

### PR DESCRIPTION
**Pull request by guilhermeltm**
Updated mpas_plot.py to account for case where infile does not contain the grid properties needed for plotting. In that case, this update offers the option to supply an additional file that does contain these properties.

I added a GTM and a comment on all relevant modifications. However, since in the pull request the modifications to the previous script are already highlighted, I did not comment out old code but simply deleted it -- let me know if there is reason for doing otherwise.

### CODE REVIEW

File: post_proc/py/plot_scalar_on_native_grid/mpas_plot.py [hunk 1]
```diff
@@ -21,6 +22,7 @@
 import cartopy.crs as ccrs
 import cartopy.feature as cfeature
 from geopy.distance import distance
+import pandas as pd
```
Comment (favba) [inline post_proc/py/plot_scalar_on_native_grid/mpas_plot.py:25]: This doesn't seem necessary, right? It seems we are not using Pandas (at least directly) anywhere...

**Review by favba — APPROVED** [review]
LGTM. I'd just remove the seemingly unnecessary `import pandas` before merging the changes. Unless I'm mistaken it is indeed necessary.
Could you verify that, @guilhermeltm, and then merge the changes?

File: post_proc/py/plot_scalar_on_native_grid/mpas_plot.py (hunk 1 above)
Comment (guilhermeltm) [reply post_proc/py/plot_scalar_on_native_grid/mpas_plot.py]: Thanks a lot, Felipe. I was planning on using pandas, and then found it to be unnecessary -- but forgot to remove the imports. In reviewing this I realized that also

import cartopy.feature as cfeature
from geopy.distance import distance

were not being used, so I removed them.

**Comment by guilhermeltm** [comment]
@favba could you please review again and merge? Since I have made a correction, I am not able to merge the code without an additional review. Thanks!

**Review by favba — APPROVED** [review]
Great!

**Pull request merged**